modules/table/src/test/java/org/apache/ignite/internal/table/distributed/gc/GcUpdateHandlerTest.java

```diff
@@ -55,20 +55,24 @@ void testVacuum() {
 
         HybridTimestamp lowWatermark = new HybridTimestamp(100, 100);
 
-        assertFalse(gcUpdateHandler.vacuum(lowWatermark));
-        verify(partitionStorage).pollForVacuum(lowWatermark);
+        assertFalse(gcUpdateHandler.vacuumBatch(lowWatermark, 1));
```

Comment:
Looks like we don't have any tests that use a batch size larger than 1

Reply:
Added a test.